lib/actions/ssh.ts

```diff
@@ -259,22 +257,16 @@ export const ssh: CommandDefinition<
 		}
 
 		// this will be a tunnelled SSH connection...
-		await exitIfNotLoggedIn();
+		await checkLoggedIn();
```

Comment:
I've used the opportunity to replace `exitIfNotLoggedIn` (which calls `process.exit`) with `checkLoggedIn` (which throws a subclass of ExpectedError), which is our current best practice (it allows errors to bubble up to a centralised error processing function in `errors.ts`, and avoids the side effects of `process.exit` such as discarding data still in buffers -- corrupt files, partial console output).